.github/workflows/rust.yml

```diff
@@ -1,6 +1,6 @@
 name: Rust
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
any reason to remove push? Adding push will cause it to build on new commits pushed to PR as well

Reply:
I do not think building twice the same thing is necessary for both PUSH and PULL_REQUEST.